hellabot.go

```diff
@@ -270,11 +270,19 @@ func (bot *Bot) Reply(m *Message, text string) {
 
 // Msg sends a message to 'who' (user or channel)
 func (bot *Bot) Msg(who, text string) {
-	for len(text) > 400 {
-		bot.Send("PRIVMSG " + who + " :" + text[:400])
-		text = text[400:]
+	for _, line := range strings.Split(text, "\n") {
```

Comment:
This won't work for windows style newlines. Use a `bufio.Scanner`.

Reply:
Thanks for the tip ^^